### ISSUE TITLE
version 1.3.0 does not return the right dynamic type

### ISSUE THREAD

**Issue by jnUser** [trenoncourt/AutoQueryable#29]
In version 1.3.0
IQueryable<object> queryResult = result.AutoQueryable(queryString) returns IQuerable of objects that are of type objectRealTypeName + long seemingly random string.

DataObjects.DomainObjects.v2.ExpenseOutcomeBrowseDO73390d5deeee3603d91efe6bc2b21610c9d73f8f556755dcc7b16ee102d25adc

Casting these objects to their needed type is not possible as the type is incorrect.
Is this a bug or has something changed? Reverted back to 1.1.0 for the moment.

I am using AutoQuerable  on .Net Core 2.0 project.

**Comment by trenoncourt** [comment]
Hi jnUser, 

Do you have a repo with an example to reproduce ? 

Do you have the same issue with 1.1.0 & EF core 2? 

Thanks.

**Comment by trenoncourt** [comment]
@jnUser 

I did some tests, and I got the random string too. But that's normal.
Version 1.1.0 worked in the same way except in one case: when the query string is empty.

AQ generates a new type dynamically so as not to keep unnecessary properties in the json (or other) generated.

I planned in the roadmap to leave the choice between auto generated type and base type.

I'll ping you when this feature becomes available
